hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -617,131 +614,135 @@ public static RaftProperties newRaftProperties(ConfigurationSource conf,
     } else if (rpc == SupportedRpcType.NETTY) {
       NettyConfigKeys.Server.setPort(properties, port);
     }
+    // For grpc set the maximum message size
+    setGrpcConfig(properties, conf);
 
     // Set Ratis storage directory
-    RaftServerConfigKeys.setStorageDir(properties,
-        Collections.singletonList(new File(ratisStorageDir)));
+    RaftServerConfigKeys.setStorageDir(properties, Collections.singletonList(new File(ratisStorageDir)));
+
+    setRaftLeaderElectionProperties(properties, conf);
+
+    setRaftLogProperties(properties, conf);
+
+    setRaftRpcProperties(properties, conf);
+
+    setRaftRetryCacheProperties(properties, conf);
+
+    setRaftNotificationProperties(properties, conf);
+
+    setRaftSnapshotProperties(properties, conf);
+
+    setRaftCloseThreshold(properties, conf);
+
+    getOMHAConfigs(conf).forEach(properties::set);
+    return properties;
+  }
+
+  private static void setGrpcConfig(RaftProperties properties, ConfigurationSource conf) {
+    final int logAppenderQueueByteLimit = (int) conf.getStorageSize(
+        OMConfigKeys.OZONE_OM_RATIS_LOG_APPENDER_QUEUE_BYTE_LIMIT,
+        OMConfigKeys.OZONE_OM_RATIS_LOG_APPENDER_QUEUE_BYTE_LIMIT_DEFAULT, StorageUnit.BYTES);
```

Comment:
Let's avoid getting the same conf twice.  We may pass it as a parameter.

Reply:
@szetszwo Thanks for the suggest and the review.patch.  I already updated new commit to try to keep less formatting changes.